docs/design/performance-benchmarking/SETUP-PERFORMANCE-SUITE.md

```diff
@@ -0,0 +1,11 @@
+# Setup Performance Testing Suite
+
+The performance testing suite is...
+
+Steps to run against meadowlark:
+
+- Configure Meadowlark and verify that it's running.
+- Load Sample data with the Invoke-LoadGrandBend or Invoke-LoadPartialGrandBend scripts.
```

Comment:
```suggestion
- Load Sample data with the Invoke-LoadGrandBend or Invoke-LoadPartialGrandBend scripts (otherwise, 
   the tests will fail without some expected descriptors and education organization Ids).
```